pkg/synopsysctl/blackduck_migrate.go

```diff
@@ -185,21 +185,22 @@ func migrate(bd *v1.Blackduck, operatorNamespace string, crdNamespace string, fl
 	return destroyOperator(operatorNamespace, crdNamespace, skipDestroyorRestartOperator)
 }
 
-// isFeatureEnabled check whether the feature is enabled by reading through the Black Duck environment variables
-func isFeatureEnabled(environs []string, featureName string, expectedValue string) bool {
-	for _, value := range environs {
+// isFeatureEnabled check whether the feature is enabled by reading through the Black Duck environment variables and also it removes its corresponding entry in environs
+func isFeatureEnabled(environs []string, featureName string, expectedValue string) (bool, []string) {
```

Comment:
I think this function should be renamed to `removeEnviron()` or it should be split into two functions:
1. `environHasCorrectValue(environs []string, featureName string, expectedValue string) bool`
2. `removeEnviron(environs []string, featureName string) []string`